README.rst

```diff
@@ -609,6 +609,39 @@ Cache can be disabled via ``--using-cache`` option or config file:
         ->setUsingCache(false)
     ;
 
+Cache file can be specified via ``--cache-file`` option or config file:
+
+.. code-block:: php
+
+    <?php
+
+    return Symfony\CS\Config\Config::create()
+        ->setCacheFile(__DIR__.'/.php_cs.cache')
+    ;
+
+Speed up builds on Travis
```

Comment:
I wouldn't recommend using PHP-CS-FIXER on travis TBH. If travis fails, you don't immediately know if it's important (ie, your tests failed), or trivial, (you have trailing whitespace).

Could we recommend https://styleci.io/?